src/pages/EditMonitor.vue

```diff
@@ -375,6 +385,16 @@
                                     </div>
                                     <br>
 
+                                    <label for="jsonPathOperator" class="form-label">{{ $t("Json Query Operator") }}</label>
+                                    <select id="jsonPathOperator" v-model="monitor.jsonPathOperator" class="form-select" required>
+                                        <option value="==">{{ $t("Json Query Equals") }} (==)</option>
+                                        <option value="!=">{{ $t("Json Query Not Equals") }} (!=)</option>
+                                        <option value="<">{{ $t("Json Query Less Than") }} (&lt;)</option>
+                                        <option value="<=">{{ $t("Json Query Less Than Or Equals") }} (&lt;=)</option>
+                                        <option value=">">{{ $t("Json Query Greater Than") }} (&gt;)</option>
+                                        <option value=">=">{{ $t("Json Query Greater Than Or Equals") }} (&gt;=)</option>
```

Comment:
I don't like about the current display that the operators and helptext semantic is switched. Please make switch their order and make sure that the opeartor is the first thing you read.

Experiment if making the helptext less dark or the operator bold is a better call.